src/main/java/org/jabref/logic/importer/util/GroupsParser.java

```diff
@@ -206,6 +206,14 @@ private static KeywordGroup keywordGroupFromString(String s, Character keywordSe
         return newGroup;
     }
 
+    private static int convertTokenToNumber(String token) {
+        try {
+            return Integer.parseInt(token);
+    	} catch (Exception e) {
+    		return -1;
```

Comment:
JabRef should really show some parsing error in the log and maybe too in the ui, even though this accepts faulty input.